Src/java/elm-fhir/src/main/java/org/cqframework/cql/elm/requirements/fhir/DataRequirementsProcessor.java

```diff
@@ -975,6 +978,19 @@ private org.hl7.fhir.r5.model.DataRequirement toDataRequirement(ElmRequirementsC
             dr.addMustSupport(s);
         }
 
+        if (pertinenceContext != null && pertinenceContext.getPertinenceValue() != null
+                && !(pertinenceContext.getPertinenceValue().trim().isEmpty())) {
+            Extension extension = new Extension();
+            extension.setUrl("http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-pertinentResource");
```

Comment:
Where did the spelling for this extension come from? I don't think "pertinentResource" is accurate, why isn't it just "pertinence"? And I don't see a related tracker supporting actually defining the extension in any specification.